tests/Feature/ImportReportTest.php

```diff
@@ -46,8 +46,6 @@ public function test_generate_csv_writes_to_path(): void
         $total = $mismatches->count();
         $pending = $mismatches->where('review_status', 'pending')->count();
 
-        $this->travelTo(now()); // stop the clock
```

Comment:
This change doesn't necessarily relate to the ticket we are working, I think it will probably be best to do this in a separate PR.